File: Luis Mera/python/Leccion13/Gerente.py
from Empleado import Empleado


class Gerente(Empleado):
    def __init__(self, departamento, nombre, sueldo):
        super().__init__(nombre, sueldo)
        self.departamento = departamento

    def __str__(self):
        return f"Nombre: {self.nombre}, Sueldo: {self.sueldo}, Departamento: {self.departamento}"

File: Lorena-Artieda/Java/Leccion04/sobreescritura/src/domain/Gerente.java
package domain;

public class Gerente extends Empleado{
    private String departamento;

    public Gerente(String nombre, double sueldo, String departamento) {
        super(nombre, sueldo);
        this.departamento = departamento;
    }

    @Override
    public String obtenerDetalle(){
        return super.obtenerDetalle()+ ", Departamento: "+ this.departamento;
    }
}
